uber/templates/fields/attendee.html

```diff
@@ -1214,8 +1214,13 @@ <h4>{{ tier.name }}</h4>
   <div class="checkbox col-sm-9 col-sm-offset-3">
     <label for="pii_consent" class="checkbox-label">
       <input type="checkbox" name="pii_consent" id="pii_consent" value="1" required {% if pii_consent %}checked{% endif %} />
-      <strong>Yes</strong>, I understand that {{ c.ORGANIZATION_NAME }} will store the personal information I provided above for the purpose of contacting me about my registration.
+      <strong>Yes</strong>, I understand and agree that {{ c.ORGANIZATION_NAME }} will store the personal information I provided above for the limited purposes of contacting me about my registration, hotel accommodations, kick-in packages, or volunteer opportunities selected at sign-up.
```

Comment:
This is the line that should be a config item so CE the list of "limited purposes" may change.